core/src/main/java/com/novoda/noplayer/internal/exoplayer/SkippedAdverts.java

```diff
@@ -11,14 +11,16 @@ private SkippedAdverts() {
         // Utility class.
     }
 
-    static AdPlaybackState from(long currentPositionInMillis, List<AdvertBreak> advertBreaks, AdPlaybackState adPlaybackState) {
+    /**
+     * Transforms all adverts that are not currently Played to Skipped.
+     *
+     * @param advertBreaks    The client representation of the adverts, our source of truth.
+     * @param adPlaybackState The {@link AdPlaybackState} to mutate with the new states.
+     * @return The {@link AdPlaybackState} with the new Skipped states.
+     */
+    static AdPlaybackState from(List<AdvertBreak> advertBreaks, AdPlaybackState adPlaybackState) {
```

Comment:
Maybe change the method name for clarity

Reply:
2d9062e

Reply:
18575bd